src/main/org/deidentifier/arx/ARXLogisticRegressionConfiguration.java

```diff
@@ -126,7 +126,7 @@ public PriorFunction getPriorFunction() {
     /**
      * @return the seed
      */
-    public long getSeed() {
+    public int getSeed() {
```

Comment:
Data types following master. Please move all common fields to the abstract base class